app/helpers/application_helper.rb

```diff
@@ -9,7 +9,9 @@ def youtube_url(source)
   #   and check for updates every few days.
   #
   def configuration
-    @configuration ||= Tmdb::Configuration.new
+    Rails.cache.fetch("tmdb/configuration", expires_in: 1.day) do
+      ConfigurationDecorator.decorate($tmdb.get('configuration').body)
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.